cibuildwheel/linux.py

```diff
@@ -92,8 +104,12 @@ def build(options: BuildOptions) -> None:
               file=sys.stderr)
         exit(2)
 
+    if options.use_binfmt:
+        log.step('Installing binfmt...')
+        subprocess.check_output(['docker', 'run', '--rm', '--privileged', 'docker/binfmt:a7996909642ee92942dcd6cff44b9b95f08dad64'])
```

Comment:
Judging by the about section and archival of the [`docker/binfmt` repo](https://github.com/docker/binfmt), maybe this should be replaced with [`linuxkit/binfmt`](https://hub.docker.com/r/linuxkit/binfmt) as per its suggestion?

Reply:
Yeah absolutely. I will push up a patch.